Assets/Scripts/SethRefactor/Testing.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ScavengerWorld
```

Comment:
Does this need to be committed?

Reply:
Yes, it's there to provide a UI that we can use to test that actions are firing off as expected.